hikari/impl/rest.py

```diff
@@ -749,6 +771,13 @@ async def _perform_request(
         if auth:
             headers[_AUTHORIZATION_HEADER] = auth
 
+        data: typing.Union[None, aiohttp.BytesPayload, aiohttp.FormData] = None
+        if json:
+            if form_builder:
+                raise ValueError("Can only provide one of 'json' or 'form_builder', not both")
```

Comment:
If you've got cases like this which aren't meant to happen and thus aren't covered by tests maybe consider putting a `# pragma: no cover` comment on it (assuming that works with hikari's tooling)

Reply:
I just forgot to add a test case for it 😅

Ty for pointing it out!

Reply:
Fixed!